spot_driver/launch/spot_driver.launch.py

```diff
@@ -122,18 +157,7 @@ def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
         raise FileNotFoundError("Configuration file '{}' does not exist!".format(config_file_path))
 
     if not mock_enable:
```

Comment:
nit: "notting" this condition is kind of pointless here.  I'd rather see a more straightforward `if mock_enable:`